client-misk-jooq/src/test/kotlin/app/cash/backfila/client/misk/WidgetCompoundKeyBackfillDbSetup.kt

```diff
@@ -0,0 +1,83 @@
+package app.cash.backfila.client.misk
+
+import app.cash.backfila.client.misk.jooq.gen.tables.references.WIDGETS
+import app.cash.backfila.client.misk.setup.CompoundKey
+import app.cash.backfila.client.misk.setup.JooqTransacter
+import misk.time.FakeClock
+import misk.tokens.TokenGenerator
+
+object WidgetCompoundKeyBackfillDbSetup {
```

Comment:
Yeah should probably just be a companion object?

Reply:
If you used my base class idea then each of these could just be a companion object on the implementations.